libkbfs/folder_block_ops.go

```diff
@@ -855,7 +864,7 @@ func (fbo *folderBlockOps) AddDirEntryInCache(lState *lockState, dir path,
 }
 
 func (fbo *folderBlockOps) removeDirEntryInCacheLocked(lState *lockState,
-	dir path, oldName string) {
+	dir path, oldName string, oldDe DirEntry) {
```

Comment:
Oh sorry, it's not used until #951.  Sorry, bad job of splitting things up.  Let me know if you want me to move its introduction until then.